### DISCUSSION TITLE
Disable inputs

### DISCUSSION THREAD

**Discussion by augustosnk12** [react-hook-form/react-hook-form#8280] [General]
Hello dear community, 

I have some question: is there a way to disable a lot of inputs? In some cases I need to disable some fields, not every time. And because there is so many inputs it's not a good thing to use the **disabled** props everytime. I would like to have a way to do this easily. 

Thanks you all in advance 😘

**Comment by Moshyfawn** [comment]
Hey, @augustosnk12!

Depending on your needs, you can wrap your inputs section with the `fieldset` element and set the `disabled` attribute
```jsx
<fieldset disabled>
  <input />
  <input />
</fieldset>
```
